src/components/CheckboxLabeled/examples/label-as-prop.jsx

```diff
@@ -9,27 +9,25 @@ const style = {
 export default createClass({
 	render() {
 		return (
-			<section>
-				<section>
-					<CheckboxLabeled Label="Just text" style={style} />
-					<CheckboxLabeled Label={<span>HTML element</span>} style={style} />
-					<CheckboxLabeled
-						Label={[
-							'Text in an array',
-							'Only the first value in the array is used',
-							'The rest of these should be ignored',
-						]}
-						style={style}
-					/>
-					<CheckboxLabeled
-						Label={[
-							<span>HTML element in an array</span>,
-							<span>Again only the first value in the array is used</span>,
-							<span>The rest should not be rendered</span>,
-						]}
-						style={style}
-					/>
-				</section>
+			<section style={{ display: 'inline-flex', flexDirection: 'column' }}>
```

Comment:
Maybe you should mention in a `<p>` tag that people should wrap this component with specific styles for optimal viewing.